cmd/test-external-check/README.md

```diff
@@ -45,3 +45,46 @@ spec:
           cpu: 10m
           memory: 50Mi
 ```
+
+### image-pull-check
+
+This is a container used to test availability of external image respositories in Kuberhealthy.  Simply place the `kuberhealthy/test-external-check` image on the repository you need tested.
+
+##### Example `khcheck`
+
+The pod is designed to attempt a pull of the test image from the remote repository (never from local) every 10 minutes. If the image is unavailable, an error will be reported to the Kuberheakthy API.
+
+To put a copy of this image to your repository, run `docker pull kuberhealthy/test-external-check` and then `docker push my.repository/repo/test-external-check`.
```

Comment:
can this comment be placed in the `### image-pull-check` section? feel like it might better fit there after `Simply place the `kuberhealthy/test-external-check` image on the repository you need tested.`